definitions/npm/react-intl_v2.x.x/flow_v0.63.x-/react-intl_v2.x.x.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * Original implementation of this file by @marudor at https://github.com/marudor/flowInterfaces
+ * Copied here based on intention to merge with flow-typed expressed here:
+ * https://github.com/marudor/flowInterfaces/issues/6
+ */
+// Mostly from https://github.com/yahoo/react-intl/wiki/API#react-intl-api
+
+import type { Element, ChildrenArray } from "react";
+
+type $npm$ReactIntl$LocaleData = {
```

Comment:
These should all be wrapped inside the module generally.

Reply:
Done. 
I think this part I just copy pasted from prev implementation.

Reply:
Makes sense, I appreciate you taking the extra time to improve the libdef!